flatcar-linux/portage-stable
# Use upstream talloc, tdb, tevent

They went to coreos-overlay when we had to include downstream patches. Use upstream packages again for https://github.com/flatcar-linux/coreos-overlay/pull/364

# How to use/Testing done

See https://github.com/flatcar-linux/coreos-overlay/pull/364

Note: Pick all patches for all channels.
